contracts/src/PDPService.sol

```diff
@@ -225,9 +230,15 @@ contract PDPService {
         return rootId;
     }
 
+    function enqueueRemovals(uint256 setId, uint256[] calldata rootIds) public {
```

Comment:
Other names

* ProposeRemove
* RegisterRemove
* Remove

Probably we don't want just `Remove` as it gives the wrong idea